src/c++/perf_analyzer/docs/examples/profile.py

```diff
@@ -175,72 +235,135 @@ def collect_periodic_latencies(args, filename):
     return bins
 
 
-def calculate_avg_periodic_latencies(args, filename):
+def calculate_avg_periodic_latencies(args, profile_result, filename):
```

Comment:
nit: Why pass an output parameter when it's only accessed once to set one output. Might as well return the output and assign it in the calling function how it's currently assigned here.

Reply:
I wanted to be consistent with other functions: `calculate_online_metrics` and `calculate_offline_metrics`.